scripts/startLocalFunctionsTestnet.js

```diff
@@ -0,0 +1,54 @@
+const process = require("process")
+const path = require("path")
+const fs = require("fs")
+const { startLocalFunctionsTestnet } = require("@chainlink/functions-toolkit")
+const { utils, Wallet } = require("ethers")
+// Loads environment variables from .env.enc file (if it exists)
+require("@chainlink/env-enc").config("../.env.enc")
+;(async () => {
+  const requestConfigPath = path.join(process.cwd(), "Functions-request-config.js") // @dev Update this to point to your desired request config file
+
+  const localFunctionsTestnetInfo = await startLocalFunctionsTestnet(
+    requestConfigPath,
+    {
+      logging: {
+        debug: false,
+        verbose: false,
+        quiet: true, // Set this to `false` to see logs from the local testnet
+      },
+    } // Ganache server options (optional)
+  )
+
+  console.log("\nFunctionsRouter Contract Address:", localFunctionsTestnetInfo.functionsRouterContract.address)
+  console.log("DON ID:".padStart(33, " "), localFunctionsTestnetInfo.donId)
+  console.log("Mock LINK Token Contract Address:", localFunctionsTestnetInfo.linkTokenContract.address, "\n")
+
+  // Fund wallets with ETH and LINK
+  const addressToFund = new Wallet(process.env["PRIVATE_KEY"]).address
+  await localFunctionsTestnetInfo.getFunds(addressToFund, {
```

Comment:
just realised that getFunds is a misleading/confusing name.  For consistency can we make this add funds?  I realise this should be in the toolkit repo! sorry I missed it there!

Reply:
Hmmmm I am not sure `addFunds` is the best name either. I'll merge this as-is for now, but maybe we should think of a better name.